xml/System/FormattableString.xml

```diff
@@ -142,10 +142,28 @@
         <Parameter Name="formattable" Type="System.FormattableString" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="formattable">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="formattable">The string to be formatted.</param>
+        <summary>Format the given object in the current culture.</summary>
+        <returns>The formatted string in the current culture.</returns>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
+
+### Remarks
+
+This static method may be imported in C# by:
+
+```cs
```

Comment:
I don't know if we support remarks with a programming language. If not, let me know and I can change this to single back-ticks.